### DISCUSSION TITLE
Event Streams Logo Icon

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1295] [Request: Icon] [answered]
Hello DSYS! 

My Engineering Team have already added the logo icon to the Paste repository. 

I'd like to request to add `product/EventStreams` icon to Figma only. 

Here's the [icon](https://library.twilio.com/media/?mediaId=30285947-F40E-4E09-AC416C073970CBCD) from the Brand team and a screenshot for reference: 

<img width="602" alt="Screen Shot 2021-03-17 at 9 43 04 AM" src="https://user-images.githubusercontent.com/65037539/111504729-2d540780-8705-11eb-924e-69560dbc1b65.png">

Thank you, 
Mindy

**Comment by serifluous** [comment, marked as answer]
Hi @mindytom ! Can you check that you have the "Paste Components" library linked in Figma, and do a search for "eventstreams" in the Assets panel?

<img width="331" alt="Screen Shot 2021-03-17 at 9 55 58 AM" src="https://user-images.githubusercontent.com/3821840/111506624-0a2a5780-8707-11eb-9eb9-614bc4b57b95.png">

If you still can't find it, feel free to drop a link to your file here, and we can help troubleshoot more.

**Reply by mindytom** [reply]
Found it! Thanks for the quick reply 🥇
